src/kernels/ipywidgets-message-coordination/cdnWidgetScriptSourceProvider.base.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
is '.base' needed in the filename?

Reply:
No. I just used it to distinguish that this is a base class for the .node and .web files. I've done this in other spots.